brave/brave-core
# Remove Rewards onboarding from the tip dialog

Resolves https://github.com/brave/brave-browser/issues/24538

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

- Open the browser with a new profile.
- Navigate to a twitter user page (e.g. `https://twitter.com/brave`)
- Wait for the "tip" icons to appear within tweets.
- Click the tip icon.
- Verify that the Brave Rewards panel is opened and the onboarding modal is displayed.
